rclpy/rclpy/__init__.py

```diff
@@ -109,6 +110,7 @@ def spin_once(node, *, timeout_sec=None):
         if request:
             response = srv.callback(request, srv.srv_type.Response())
             srv.send_response(response, header)
+    gc.collect()
```

Comment:
I don't think that we should explicitly call `gc.collect` - especially not in a function like `spinOnce` which is called in a potentially very high frequency.